source/distributed-pick.d.ts

```diff
@@ -80,6 +80,4 @@ if (pickedUnion.discriminant === 'A') {
 @category Object
 */
 export type DistributedPick<ObjectType, KeyType extends KeysOfUnion<ObjectType>> =
-	ObjectType extends unknown
```

Comment:
This conditional is no longer required, because the implementation now uses a homomorphic mapped types, which already distributes over union types.